apps/wallet/src/engine/runners/solana/index.ts

```diff
@@ -77,7 +77,7 @@ export const createSolanaRunner: CreateFunction = async (config) => {
 					}),
 					watchLogs(connection, cluster, wallet),
 					watchAccount(connection, cluster, wallet, wallet),
-					getTransactionsHistory(connection, cluster, wallet, accounts),
+					getTransactionsHistory(connection, wallet, accounts),
```

Comment:
You might need to add cluster for the history, in the future, we might support both devnet and mainnet, so keep the cluster might be able to filter out tokens/nfts/history